samples/connectedhomeip/light_bulb/README.rst

```diff
@@ -1,19 +1,20 @@
+.. _matter_light_bulb_sample:
 .. _chip_light_bulb_sample:
```

Comment:
Remove the old label?

Reply:
Let's keep it until we completely switch to the new naming convention.